resources/assets/scripts/app/directives/href-loader.coffee

```diff
@@ -0,0 +1,36 @@
+# Pony.fm - A community for pony fan music.
+# Copyright (C) 2015 Peter Deltchev
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+angular.module('ponyfm').directive 'pfmHrefLoader', () ->
```

Comment:
What's the intended purpose of this directive? As far as I can tell, it replaces its own `pfm-href-loader` attribute with an `href` tag - but why not use [`ng-href`](https://docs.angularjs.org/api/ng/directive/ngHref) instead?